CorsixTH/Lua/dialogs/place_staff.lua

```diff
@@ -75,17 +75,39 @@ function UIPlaceStaff:onCursorWorldPositionChange(x, y)
   self.tile_y = math_floor(y)
 end
 
+-- Check the given tile and the four around it for (the centre of) a reception desk.
+local function nearReceptionDesk(world, x, y)
+  return world:getObject(x, y, "reception_desk") or
```

Comment:
Is there a better way of representing this? If not then that's fine too.

Reply:
It could be made generally useful by reading the footprint of objects, but if this is the only use for it then it would be harder to read.

Reply:
Just add a function that searches a specified area for an object? (It sounds like we may already something like that, but don't know.)

Reply:
There are also searches for the opposite part of the entrance door, maybe there are more

Reply:
Changed to using two world functions together.